python/knowhere/knowhere.i

```diff
@@ -354,4 +356,51 @@ DumpRangeResultDis(knowhere::DataSetPtr result, float* dis, int len) {
     }
 }
 
+void
+Dump(knowhere::BinarySetPtr binset, const std::string& file_name) {
+    auto binary_set = *binset;
+    auto binary_map = binset -> binary_map_;
+    std::ofstream outfile;
+    outfile.open(file_name, std::ios::out | std::ios::trunc);
+    for (auto it = binary_map.begin(); it != binary_map.end(); ++it) {
+        // serialization: name_length(size_t); name(char[]); binset_size(size_t); binset(uint8[]); 
+        auto name = it->first;
+        outfile << name.size();
+        outfile << name;
```

Comment:
name.size() is type `size_t` and is platform dependent, please write sizes as `int64_t`

Reply:
fixed